SygicMapsKit/Classes/NavigationModule/SYMKNavigationView.swift

```diff
@@ -59,6 +64,14 @@ public class SYMKNavigationView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
     
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        
+        let isLandscape = SYUIDeviceOrientationUtils.isLandscape()
+        trailingInstructionAnchor?.isActive = !isLandscape
+        landscapeInstructionWidthAnchor?.isActive = isLandscape
```

Comment:
zrobil by som metodku:
func updateInstructionViewConstriants(for landscape: Bool)